esbuild/assets.config.js

```diff
@@ -31,6 +32,12 @@ const buildAssets = buildConfig => {
       clean({
         patterns: glob.sync(buildConfig.assets.clear),
       }),
+      manifestPlugin({
+        generate: entries =>
+          Object.fromEntries(
+            Object.entries(entries).map(([key, value]) => [key.replace('dist', ''), value.replace('dist', '')]),
+          ),
```

Comment:
i have a sneaking suspicion that [options.shortNames](https://github.com/jfortunato/esbuild-plugin-manifest?tab=readme-ov-file#optionsshortnames) might achieve the same result, but more clearly. did you try it?

Reply:
i take that back. i tried all combos of `shortNames` and they either strip too much or not enough.

so you might as well keep this `generate` rewrite instead. a potential suggestion: "dist" would be replaced everywhere in the path so could feasibly remove something unexpected. maybe use `/^dist\//` instead??